### PR TITLE
added tabinfo plugin

### PR DESCRIPTION
**Pull request by dakhnod**
This plugin displays information about the current print in the browser tab title

preview

### CODE REVIEW

File: _plugins/tabinfo.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+tags:
+- information
+- progress
+- tab
+- browser
+- overview
+
+screenshots:
+- url: https://github.com/dakhnod/OctoPrint-TabInfo/assets/26143255/0cfb75bb-41e6-4b7e-94f7-dc35010727c5
```
Comment (jneilliii) [inline _plugins/tabinfo.md]: include the images directly in the PR in the path /assets/img/plugins/tabinfo/ and update links to relative path including file name.

File: _plugins/tabinfo.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  alt: plugin preview in browser
+  caption: plugin preview in browser
+
+featuredimage: https://github.com/dakhnod/OctoPrint-TabInfo/assets/26143255/0cfb75bb-41e6-4b7e-94f7-dc35010727c5
```
Comment (jneilliii) [inline _plugins/tabinfo.md]: include the images directly in the PR in the path /assets/img/plugins/tabinfo/ and update links to relative path including file name.

**Review by jneilliii — CHANGES_REQUESTED** [review]
Just need to add the images to the PR and adjust URLs.

**Comment by dakhnod** [comment]
@jneilliii done, thanks

**Review by jneilliii — APPROVED** [review]

**Pull request merged**